### PR TITLE
Fixed QUnit tests - cross-domain - run-test (part of #471)

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @AlexanderMoskovkin, @helen-dikareva

### CODE REVIEW

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
It looks the target test is ok. So, LGTM

**Comment by AlexanderMoskovkin** [comment]
BTW, it seems we have another unstable test now: `OS X 10.11 chrome  - /fixtures/core/dom-utils-test.js
Test: isIFrameWindowInDOM`

**Comment by helen-dikareva** [comment]
lgtm

File: test/client/fixtures/runner/cross-domain/run-test.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 var RunnerBase     = testCafeRunner.get('./runner-base');
 
 hhsettings.serviceMsgUrl = '/ping/10';
 
+// HACK: we shouldn't override native dialogs methods, while testing on the
+// android simulator, because this results in an unexpected window.alert call.
+// https://github.com/DevExpress/testcafe/issues/471#issuecomment-220386269.
+if (browserUtils.isAndroid) {
+    RunnerBase.prototype._initNativeDialogs = function () {
+
```
Comment (AlexanderMoskovkin) [inline test/client/fixtures/runner/cross-domain/run-test.js]: unnecessary empty line.
Just remove it and I will merge the PR

**Comment by georgiy-abbasov** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**